configure.ac

```diff
@@ -1202,6 +1202,14 @@ if test x$MULTIARCH != x; then
 fi
 AC_SUBST([MULTIARCH_CPPFLAGS])
 
+# Guess C stack direction
+AS_CASE([$host],
+  [hppa-*|hppa64-*], [_Py_STACK_GROWS_DOWN=0],
```

Comment:
In Gentoo, we use `hppa(1.1|2.0)-unknown-linux-gnu`. Maybe change it to just `hppa*`?